onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -107,14 +122,29 @@ def quantize_tensors(self):
             # Quantize the input
             initializer = find_by_name(tensor_name, self.model.initializer())
             if initializer is not None:
-                q_weight_name, zp_name, scale_name = self.quantize_weight(initializer, self.weight_qType)
 
-                inputs = [q_weight_name, scale_name, zp_name]
-                output_name = tensor_name + '_DequantizeLinear'
-                node = onnx.helper.make_node("DequantizeLinear", inputs, [output_name],
-                                             tensor_name + '_DequantizeLinear')
-                self.model.add_node(node)
-                self.model.replace_input_of_all_nodes(tensor_name, tensor_name + "_DequantizeLinear")
+                if self.force_qdq_appear_as_pair:
```

Comment:
Will add_qdq_pair_to_weight and  keep_float_weight be more meaningful than force_qdq_appear_as_pair and add_weight_initializer? or combine them to one? Both purposes are to apply QDQ pair to floating-point weight.

Reply:
Agree that add_qdq_pair_to_weight and keep_float_weight are more meaningful. Will modify them.